lib/typescript/render/providers/facebook/FacebookRender.tsx

```diff
@@ -139,14 +139,14 @@ function facebookOutbound(message: Message): ContentUnion {
     };
   }
 
-  if (messageJson.attachment || messageJson.message?.attachments) {
```

Comment:
This one I am a bit confused about: How would the `RenderedContent` abstraction change the structure of the `content` field? 🤔

Reply:
the `RenderedContent` abstraction does not change the structure of the content field: when we render templates, there is always going to be a message field inside content (whereas it's not always the case for messages)
 
I've just refactored the code so that we don't always have to check for `messageJson.message`. I've added `const messageJson = message.content.message ?? message.content;` (above in the file) to check if the content field contains `message.content.message`

Reply:
Ah, right I missed that!